### PR TITLE
Add missing step for nginx guide.

### LINKED ISSUE #1245: Chatty nginx ingress logs
**What should be cleaned up or changed**:

I followed instructions [here](https://kind.sigs.k8s.io/docs/user/ingress/) on how to setup nginx ingress. But when I run:

```
kubectl logs --namespace ingress-nginx nginx-ingress-controller-6876b49448-svzvl
```

I see many lines like:

```
W0111 08:08:11.996211       7 queue.go:130] requeuing &ObjectMeta{Name:sync status,GenerateName:,Namespace:,SelfLink:,UID:,ResourceVersion:,Generation:0,CreationTimestamp:0001-01-01 00:00:00 +0000 UTC,DeletionTimestamp:<nil>,DeletionGracePeriodSeconds:nil,Labels:map[string]string{},Annotations:map[string]string{},OwnerReferences:[]OwnerReference{},Finalizers:[],ClusterName:,ManagedFields:[]ManagedFieldsEntry{},}, err services "ingress-nginx" not found
```

Looking [here](https://kubernetes.github.io/ingress-nginx/deploy/#bare-metal) I run the following:

```
kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.26.2/deploy/static/provider/baremetal/service-nodeport.yaml
```

And now there are no such messages anymore. Maybe it should be added to documentation?

**Why is this needed**:

It seems it is just not fully configured.

### PR DESCRIPTION
**Pull request by amwat**
fixes #1245

### CODE REVIEW

File: site/content/docs/user/ingress.md [hunk 1]
```diff
@@ -55,12 +55,18 @@ nodes:
 EOF
 {{< /codeFromInline >}}
 
-Apply the [mandatory ingress-nginx components](https://kubernetes.github.io/ingress-nginx/deploy/#prerequisite-generic-deployment-command).
+Apply the [mandatory ingress-nginx components](https://kubernetes.github.io/ingress-nginx/deploy/#prerequisite-generic-deployment-command) 
 
 {{< codeFromInline lang="bash" >}}
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml
```
Comment (aledbf) [inline site/content/docs/user/ingress.md]: Please also change this to use 0.27.0

Comment (amwat) [reply site/content/docs/user/ingress.md]: done

**Comment by amwat** [comment]
https://deploy-preview-1251--k8s-kind.netlify.com/docs/user/ingress/#ingress-nginx

**Comment by BenTheElder** [comment]
/approve
thanks!!

**Pull request merged**